File: src/main/java/com/tm/dto/bean/task/TaskDeleteRequestDto.java
package com.tm.dto.bean.task;

import io.swagger.annotations.ApiModelProperty;

/**
 * タスク削除リクエストDTO。
 */
public class TaskDeleteRequestDto {

    /**
     * タスクID
     */
    @ApiModelProperty(value="${TaskDeleteRequestDto.taskId}")
    private String taskId;

    public String getTaskId() {
        return taskId;
    }
    public void setTaskId(String taskId) {
        this.taskId = taskId;
    }
}

File: src/main/kotlin/com/tm/dto/bean/TaskDeleteRequestDto.kt
package com.tm.dto.bean

import io.swagger.annotations.ApiModelProperty

/**
 * タスク削除リクエストDTOクラス。
 */
data class TaskDeleteRequestDto (

        /**
         * タスクID
         */
        @ApiModelProperty(value = "\${TaskDeleteRequestDto.taskId}")
        var taskId: String
)